pyi.py

```diff
@@ -752,26 +752,6 @@ def _analyze_base_node(
     {"math.inf", "math.nan", "math.e", "math.pi", "math.tau"}
 )
 
-_ALLOWED_ATTRIBUTES_IN_DEFAULTS = frozenset(
-    {
-        "sys.base_prefix",
-        "sys.byteorder",
-        "sys.exec_prefix",
-        "sys.executable",
-        "sys.hexversion",
-        "sys.maxsize",
-        "sys.platform",
-        "sys.prefix",
-        "sys.stdin",
-        "sys.stdout",
-        "sys.stderr",
-        "sys.version",
-        "sys.version_info",
-        "sys.winver",
-        "_typeshed.sentinel",
-    }
-)
-
 _ALLOWED_SIMPLE_ATTRIBUTES_IN_DEFAULTS = frozenset({"sentinel"})
```

Comment:
and this one?

Reply:
They are still used in `_is_valid_default_value_with_annotation`, I think this PR is incomplete, I'll have a look.

Reply:
This is misnamed. It's not about attributes, but names. I've renamed it.